src/components/Pricing/PricingPro.js

```diff
@@ -8,8 +8,24 @@ import PriceCardFour from "./pricing-pro-cards/PriceCardFour";
 const PricingPro = () => {
   const [price, setPrice] = useState({});
 
-  const monthly = { type: "monthly", pro100: 29, pro500: 79, pro2k: 179 };
-  const annual = { type: "annual", pro100: 24, pro500: 66, pro2k: 149 };
+  const monthly = {
+    type: "monthly",
+    pro100: 29,
+    pro500: 79,
+    pro2k: 179,
+    adsPro100: 48,
```

Comment:
16 + 29 should not give you **48** on toggle as addOns but **45** compare [sellbrite](https://www.sellbrite.com/pricing-pro/) which has 19 + 29 = 48 on toggle.